common/app/navigation/helpers/UrlHelpers.scala

```diff
@@ -90,10 +88,10 @@ object UrlHelpers {
     }
 
     val acquisitionData = Json.obj(
-      "source" -> AcquisitionSource.GuardianWeb.originalName,
+      "source" -> "GUARDIAN_WEB",
```

Comment:
perhaps we could have a comment explaining that this has to correspond to a Thrift enum and linking to https://dashboard.ophan.co.uk/docs/thrift/acquisition.html#Enum_AcquisitionSource?

Reply:
Sure, I'll add the comments